include/pybind11/embed.h

```diff
@@ -198,9 +198,11 @@ inline void initialize_interpreter(bool init_signal_handlers = true,
         init_signal_handlers, argc, argv, add_program_dir_to_path);
 #else
     PyConfig config;
-    PyConfig_InitIsolatedConfig(&config);
-    config.isolated = 0;
-    config.use_environment = 1;
+    PyConfig_InitPythonConfig(&config);
+    // Don't parse arguments the way regular python parses command line arguments, because we set
+    // the arguments manually in initialize_interpreter().
+    config.parse_argv = 0;
```

Comment:
I don't really know if this comment is valid for what this field does, but this change seems to allow this to work with `PyConfig_InitPythonConfig`.

My main concern would be that the [configuration docs](https://docs.python.org/dev/c-api/init_config.html#python-initialization-configuration) say that the isolated configuration is for embedding python into an application, which is usually the use case for this function...

If that's true, then technically it would be better practice to use the isolated configuration. But since that seems to result in problems with `PYTHONPATH`, I think this solution is good. This technically seems like a more valid solution than the current state, since it's contradictory to set `config.isolated` to 0 in an isolated configuration.